contents/docs/cdp/memphis-exporter.md

```diff
@@ -1,12 +1,13 @@
 ---
 title: Memphis
 github: https://github.com/PostHog/posthog-memphisdev-app
-installUrl: https://app.posthog.com/project/apps?name=memphis
 thumbnail: ../../cdp/thumbnails/memphis_logo.jpg
 tags:
     - memphis
 ---
 
+> This app is currently unavailable while we do [some maintenance](https://github.com/PostHog/posthog/issues/15997). It will be back again soon!
```

Comment:
```suggestion
> This app is currently unavailable while we develop [a new export system](https://github.com/PostHog/posthog/issues/15997). It will be back again soon!
```